crates/viewer/re_view_graph/src/view.rs

```diff
@@ -105,9 +105,11 @@ impl ViewClass for GraphView {
             .get(&NodeVisualizer::identifier())
         {
             ViewSpawnHeuristics::new(maybe_visualizable.iter().cloned().filter_map(|entity| {
-                suggested_filter
-                    .matches(&entity)
-                    .then_some(RecommendedView::new_single_entity(entity))
+                if !suggested_filter.matches(&entity) {
+                    Some(RecommendedView::new_single_entity(entity))
+                } else {
+                    None
+                }
```

Comment:
Wow, that is very confusing naming. `suggested_filter` should be called `ignored_entities`

Reply:
Cleaned this up in https://github.com/rerun-io/rerun/pull/9788